app/views/templates/_template.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td class="question"
```

Comment:
Classes in slim can be wrote like `td.question` but it's not mistake (: Here goes next question, where have you described this class?